Tasks/AzureAppServiceSettingsV1/Strings/en-US/resources.resjson

```diff
@@ -0,0 +1,192 @@
+{
+  "loc.friendlyName": "Azure Web App Settings",
+  "loc.helpMarkDown": "[Learn more about this task](https://aka.ms/azurewebappdeployreadme)",
+  "loc.description": "Update/Add App settings an Azure Web App for Linux or Windows",
+  "loc.instanceNameFormat": "Azure Web App Settings: $(appName)",
```

Comment:
delete all resources files and build task to generate only relevant strings. then push.